api/millenium_patron.py

```diff
@@ -79,8 +79,14 @@ class MilleniumPatronAPI(BasicAuthenticationProvider, XMLParser):
           ],
           "default": "true"
         },
-        { "key": BLOCK_TYPES, "label": _("Block types"), "optional": True },
-        { "key": IDENTIFIER_BLACKLIST, "label": _("Identifier Blacklist"), "optional": True },
+        { "key": BLOCK_TYPES, "label": _("Block types"),
+          "description": _("Values of MBLOCK[p56] which mean a patron is blocked. By default, any value other than '-' indicates a block."),
+          "optional": True },
+        { "key": IDENTIFIER_BLACKLIST, "label": _("Identifier Blacklist"),
+          "type": "list",
+          "description": _("Identifiers containing any of these strings are ignored when finding the 'correct' " +
+                           "identifier for a patron's record, even if it means they end up with no identifier at all."),
```

Comment:
Add "If librarians invalidate library cards by adding strings like "EXPIRED" or "INVALID" on to the beginning of the card number, put those strings here so the Circulation Manager knows they do not represent real card numbers."